tests/test_request.py

```diff
@@ -37,7 +39,7 @@ def test__init__(self):
         assert req.user is None
 
 
-class TestRequest:
+class TestRequest:  # pylint: disable=attribute-defined-outside-init
```

Comment:
The same here, should be put under `setup_method` and `teardown_method`, not for the whole class 😉

Reply:
It would be good to standardize that. I didn't point it everywhere but if you like I can spend some time writing additional comments about this issue 🙂